src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,276 @@ redirects:
+
+    Policies can hold multiple conditions or just one. In our case, if we're looking to make sure we learn about web transaction latency we have a few options. 
+    
+    First, we could create a policy that only attaches a single issue per policy, our default option. One issue per policy reduces noise but also requires immediate action. But this means that if we've multiple conditions attached to this policy, not just this one, **Response time: Example app**, then no matter what, **all incidents in this policy will be grouped into one single issue**.
+
+    Or we could create one issue per condition. This means that any time the **Response time: Example app** condition opens an incident, all those incidents will be rolled into one issue that is connected to our condition. For our specific use case, this is the option we're going to choose because it meets our primary goal: to learn about any latency issues with web transaction time. 
+
+    Or we could create an issue for every incident. This option is the most noisy but can work well if you like to send information to an external system. 
+
+    Learn more about how to create policies [here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/).
+  </Collapser>
+</CollapserGroup>
+
+## Additional settings [#additional-settings]
+
+<CollapserGroup>
+  <Collapser
+    id="close-open-incidents"
+    title="Close open incidents"
+  >
+  An incident will automatically close when the targeted signal returns to a non-violating state for the time period indicated in the condition's thresholds. This wait time is called the recovery period.
+
+  For example: If the violating behavior is "web transaction time is longer than .50 seconds at least once in 5 minutes," then the incident will automatically close when web transaction time is equal to or lower than .50 for 5 consecutive minutes. 
+
+  When an incident closes automatically:
+
+  1. The closing timestamp is backdated to the start of the recovery period.
+  2. The evaluation resets and restarts from when the previous incident ended.
+
+  All conditions have an incident time limit setting that will automatically force-close a long-lasting incident.
+
+  We automatically default to 3 days and recommend that you use our default settings for your first alert. 
+  </Collapser>
+
+  <Collapser
+    id="custom-incident-description"
+    title="Send a custom incident description"
+  >
+  
+  Since we're creating an alert condition that lets us know if there are any latency issues with web transaction time, we want to make sure our developers have all the information they need when they are notified about this incident. We're going to use workflows to notify a team Slack channel when an incident is created and here is where we're going to define the message that is sent to the Slack channel. 
```

Comment:
```suggestion
  Since we're creating an alert condition that lets us know if there are any latency issues with web transaction time, we want to make sure our developers have all the information they need when they are notified about this incident. We're going to use workflows to notify a team Slack channel when an incident is created and define the message that is sent to the Slack channel. 
```